src/main/java/org/gridsuite/study/server/service/NetworkModificationTreeService.java

```diff
@@ -572,7 +563,7 @@ public String getSuffixedNodeName(UUID studyUuid, String nodeName) {
         return uniqueName;
     }
 
-    @Transactional(readOnly = false)
+    @Transactional
```

Comment:
?

Reply:
default value is false
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/transaction/annotation/Transactional.html